src/headerr/Header.tsx

```diff
@@ -1,40 +1,46 @@
 import { Link } from "react-router-dom";
-import { AppBar, Toolbar, Typography, Button, Box } from "@mui/material";
+import { AppBar, Toolbar, Button, Box } from "@mui/material";
 import { useMetaMask } from "../contexts/MetaMaskContext";
+import styled from "styled-components";
 
-const Header = () => {
-  const { isMetamaskConnected, connectToMetamask, disconnectFromMetamask } =
-    useMetaMask();
+const StyledAppBar = styled(AppBar)`
+  && {
```

Comment:
what does && do here?

Reply:
Without it position of my button is changed